projects/dojoswap/index.js

```diff
@@ -1,11 +1,38 @@
+const BigNumber = require("bignumber.js");
 const { getFactoryTvl } = require("../terraswap/factoryTvl");
 
 const factory = {
   classic: "inj1pc2vxcmnyzawnwkf03n2ggvt997avtuwagqngk",
 };
 
+async function staking() {
+
+  /// POOLS
+  const response = await fetch('https://analytics.dojo.trading/dashboard/pools/tvl');
+  const data = await response.json();
+  const keys = Object.keys(data);
+  const tvl = keys.reduce((tvl, key) => tvl + data[key].tvl, 0);
+
+  /// LSD
+  const response2 = await fetch('https://analytics.dojo.trading/dashboard/ldp');
```

Comment:
is there some way to fetch total amount of staked INJ on chain? 
I think it is better if we create a  new folder/listing for it, called maybe "dojoswap-lsd"

Reply:
@g1nt0ki Changes have been done